elasticsearch/manifests/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - elasticsearch.yaml
+
+patches:
+  # Replace the env variables containing the hostname with namespace
+  - path: fix-hosts.yaml
+    target:
```

Comment:
I don't think you need to specify the target in here, as it is picked up from the yaml file

Reply:
Sorry, didn't noticed your comment and merged already :sweat_smile:

Reply:
no worries